searchcore/src/tests/grouping/grouping_test.cpp

```diff
@@ -94,16 +94,17 @@ using GroupingList = GroupingContext::GroupingList;
 
 SessionId createSessionId(const std::string & s) {
     std::vector<char> vec;
-    for (size_t i = 0; i < s.size(); i++) {
-        vec.push_back(s[i]);
+    for (char c : s) {
+        vec.push_back(c);
     }
-    return SessionId(&vec[0], vec.size());
+    return {&vec[0], vec.size()};
```

Comment:
Not anything changed by this PR, but: should this be `vec.data()` instead of `vec[0]` to avoid undefined behavior if the input string is empty?

Reply:
Yes, it should. Done